cells/std/nixago/editorconfig.nix

```diff
@@ -2,10 +2,19 @@
   inputs,
   cell,
 }: let
+  l = nixpkgs.lib // builtins;
   inherit (inputs) nixpkgs;
 in {
   configData = {};
   output = ".editorconfig";
-  format = "ini";
+  engine = request: let
+    inherit (request) configData output;
+    name = l.baseNameOf output;
+    value = {
+      globalSection = {root = configData.root or false;};
```

Comment:
If not set, defaults explicitly to `false` which is cool, I guess.

Reply:
I think `true` would be a better default here.

In my experience across many repos, a proverbial 98% of them only have a single `.editorconfig` file at the project root, where `root = true` would apply. Even in monorepos including a few different languages and sub-projects with their own distinct scope, each of these scopes will likely adhere to the same coding standards declared in the root-level `.editorconfig`. So, in those proverbial 2% of projects where we've had multiple of these files, the additional files turned out to be an unnecessary source of confusion in the end.

Reply:
of course, all that said, i don't recall ever having noticed a difference with having `root = true` or `root = false` or omitting it entirely 🤷‍♀️